gulpfile.js

```diff
@@ -39,7 +39,11 @@ gulp.task('release-specs', ['prepare-cesium'], function() {
     return build(specJSName, glob.sync(testGlob), true);
 });
 
-gulp.task('release', ['release-specs']);
+gulp.task('make-schema', function(done) {
+    child_exec('npm run make-schema', undefined, done);
```

Comment:
Ok, it was a little weird. Reworked. (gen-schema already has a CommonJS module, `generateSchema.js`).